ml-algorithms/src/main/resources/mappings/ml-agent.json

```diff
@@ -0,0 +1,45 @@
+{
+  "_meta": {
+    "schema_version": 2
+  },
+  "properties": {
```

Comment:
This mappings file should be in the common module. What is the incentive to move them under ml-algorithms?

Reply:
Year I think it makes more sense to have them in the commons so any other component can just pull the commons module to access the mappings.